tests/ssl-opt.sh

```diff
@@ -1414,6 +1430,22 @@ run_test    "Renegotiation: server-initiated" \
             -s "=> renegotiate" \
             -s "write hello request"
 
+# Checks if the server parses the SigAlgs Extension. Valid only if an MD
```

Comment:
Should this test be placed immediately after the client ("Renegotiation: Signature Algorithms regression, client") one? Why not?

Reply:
I was grouping them to other renegotiation tests that are initiated by the same peer. But it seems that the tests around them are grouped by test type, so you are right that it may be better here to.